PlanningPoker.Tests/RoomsControllerTests.cs

```diff
@@ -21,32 +22,74 @@ public RoomsControllerTests(CustomWebApplicationFactory factory)
         [Fact]
         public async Task Post_Rooms()
         {
+            //Arrange
             var client = Factory.CreateClient();
-            var room = new Room() {id = "1", CreatorId = "2", name = "Room"};
+            var room = new Room() { Id = "1", CreatorName = "2", Name = "Room" };
             string serializedObject = JsonConvert.SerializeObject(room);
             var content = new StringContent(serializedObject);
+            //Act
             await client.PostAsync("api/Rooms", content);
             var response = await client.GetAsync("api/Rooms");
+            //Assert
             response.EnsureSuccessStatusCode();
         }
         [Fact]
-        public async Task Get_Users_For_Room_Where_Nothing_Exist()
+        public async Task Reset_Votes_For_Room()
         {
+            //Arrange
             var client = Factory.CreateClient();
-            var room = new Room() { id = "1", CreatorId = "2", name = "Room" };
-            string serializedObject = JsonConvert.SerializeObject(room);
-            var content = new StringContent(serializedObject);
-            await client.PostAsync("api/Rooms", content);
-            var response = await client.GetAsync("api/Rooms/1/Users");
-            Assert.Equal(HttpStatusCode.InternalServerError,response.StatusCode);
+            //Act
+            var response = await client.DeleteAsync("api/Rooms/1/Votes");
+            //Assert
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
         }
+
         [Fact]
-        public async  Task Post_Null_Rooms()
+        public async Task Post_Null_Rooms()
         {
+            //Arrange
             var client = Factory.CreateClient();
+            //Act
             var code = await client.PostAsync("api/Rooms", null);
-            Assert.Equal(expected: HttpStatusCode.BadRequest,actual: code.StatusCode);
+            //Assert
+            Assert.Equal(expected: HttpStatusCode.BadRequest, actual: code.StatusCode);
+        }
+
+        [Fact]
+        public async Task Get_Users_For_Room()
+        {
+            //Arrange
+            var client = Factory.CreateClient();
+            //Act
+            var code = await client.GetAsync("api/Rooms/1/users");
+            //Assert
+            Assert.Equal(expected: HttpStatusCode.OK, actual: code.StatusCode);
+        }
+
+        [Fact]
+        public async Task Add_User_To_Room()
+        {
+            //Arrange
+            var client = Factory.CreateClient();
+            var user = new User()
+            {
+                Id = 1,
+                Name = "Room",
+                RoomId = "1",
+                Vote = 1,
+                ConnectionId = "1",
+                Email = "123",
+                Password = ""
+            };
+            string serializedObject = JsonConvert.SerializeObject(user);
```

Comment:
string serializedObject = JsonConvert.SerializeObject(user);
var content = new StringContent(serializedObject);

Can be extracted to extension method